lib/libcheri_caprevoke/Makefile

```diff
@@ -0,0 +1,15 @@
+SRCS= libcheri_caprevoke.c
+
+NEED_COMPAT=64C
```

Comment:
Huh? NEED_COMPAT is for programs not libraries, but you don't include bsd.compat.mk anyway so it's presumably unused...

Reply:
Probably stale from prior rampaging through the build system.  I'll remove it.